subsys/nrf_security/src/drivers/cracen/cracenpsa/src/cracen.c

```diff
@@ -23,7 +24,13 @@
 
 static int users;
 
-K_MUTEX_DEFINE(cracen_mutex);
+
+#if defined(CONFIG_MULTITHREADING) && !defined(__NRF_TFM__)
+K_MUTEX_DEFINE(k_cracen_mutex);
+nrf_security_mutex_t cracen_mutex = { .zephyr_mutex = &k_cracen_mutex };
+#else
+nrf_security_mutex_t cracen_mutex;
+#endif
```

Comment:
```suggestion
NRF_SECURITY_MUTEX_DEFINE(cracen_mutex);
```

If we define a macro like this and use it here, then users of the API won't need to know API details and it will therefore be more maintainable.

It could be defined something like this:

```
#if defined(CONFIG_MULTITHREADING) && !defined(__NRF_TFM__)
#define NRF_SECURITY_MUTEX_DEFINE(arg) K_MUTEX_DEFINE(k_##arg); nrf_security_mutex_t arg = { .zephyr_mutex = &k_##arg };
#else
nrf_security_mutex_t arg;
#endif
```

Reply:
I added this for both the events and mutexes